designs/requirement/plugins/org.obeonetwork.dsl.requirement.design/src/org/obeonetwork/dsl/requirement/design/services/RequirementDecoratorBlacklist.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2008, 2021 Obeo.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.obeonetwork.dsl.requirement.design.services;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.runtime.IExtension;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.sirius.diagram.DDiagramElement;
+import org.eclipse.sirius.diagram.business.api.query.DDiagramElementQuery;
+import org.eclipse.sirius.ext.base.Option;
+import org.eclipse.sirius.viewpoint.DSemanticDecorator;
+
+/**
+ * 
+ * @author <a href="thibault.beziers-la-fosse@obeo.fr">Thibault Béziers la Fosse</a>
+ *
+ */
+public class RequirementDecoratorBlacklist {
+	
+	private Set<String> blacklistedSemanticDecoratorLabels; 
+	private static final String BLACKLIST_SEMANTIC_DECORATOR_EXTENSION_ID = "org.obeonetwork.dsl.requirement.design.blacklistedRequirementDecorator";
+	private static final String SEMANTIC_DECORATOR_NAME_ATTRIBUTE = "name";
+	
+	public RequirementDecoratorBlacklist() {					
```

Comment:
I didn't see where this constructor is called :/

Reply:
It is defined as a static attribute in the RequirementServices class, and initialized there

Reply:
ok thanks :)